woocommerce/woocommerce-ios
# [Blaze] Storage: Entity to store brief Blaze campaign info

Part of: #12120 


## Description

Adds a new entity to the storage layer to store brief blaze campaign list info received from v1.1. endpoint.


## Testing instructions
- Install a build from https://github.com/woocommerce/woocommerce-ios/pull/12121 and replace that with a build from this branch.
- The app should launch and work without any core data migration crashes. 

## Screenshots
NA

---
